symbolic/src/Data/Macaw/Symbolic/Testing.hs

```diff
@@ -437,11 +432,56 @@ data ResultExtractor sym arch where
                       -> IO a)
                   -> ResultExtractor sym arch
 
+resolveAbsoluteAddress
+  :: (MM.MemWidth w)
+  => MM.Memory w
+  -> MM.MemWord w
+  -> Maybe (MM.MemSegmentOff w)
+resolveAbsoluteAddress mem addr =
+  listToMaybe [ segOff
+              | seg <- MM.memSegments mem
+              , let region = MM.segmentBase seg
+              , Just segOff <- return (MM.resolveRegionOff mem region addr)
+              ]
```

Comment:
Note to self: if [`macaw-loader` is merged into `macaw`](https://github.com/GaloisInc/macaw/issues/283), we should consolidate this use of `resolveAbsoluteAddress` with the one currently defined in `macaw-loader`. (See GaloisInc/macaw-loader#11.)